src/server/scripts/Commands/cs_rbac.cpp

```diff
@@ -0,0 +1,375 @@
+/*
+ * Copyright (C) 2016+     AzerothCore <www.azerothcore.org>, released under GNU GPL v2 license, you may redistribute it and/or modify it under version 2 of the License, or (at your option), any later version.
+ * Copyright (C) 2008-2021 TrinityCore <http://www.trinitycore.org/>
+ */
+
+/* ScriptData
+Name: rbac_commandscript
+%Complete: 100
+Comment: All role based access control related commands (including account related)
+Category: commandscripts
+EndScriptData */
+
+#include "ScriptMgr.h"
+#include "AccountMgr.h"
+#include "Chat.h"
+#include "Config.h"
+#include "Language.h"
+#include "Player.h"
+#include "Realm.h"
+#include "World.h"
+#include "WorldSession.h"
+
+#if ACORE_COMPILER == ACORE_COMPILER_GNU
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+#endif
+
+struct RBACCommandData
+{
+    RBACCommandData(): id(0), realmId(0), rbac(nullptr), needDelete(false) { }
+    ~RBACCommandData()
+    {
+        if (needDelete)
+            delete rbac;
+    }
+
+    uint32 id;
+    int32 realmId;
+    rbac::RBACData* rbac;
+    bool needDelete;
+};
+
+class rbac_commandscript : public CommandScript
+{
+public:
+    rbac_commandscript() : CommandScript("rbac_commandscript") { }
+
+    std::vector<ChatCommand> GetCommands() const override
+    {
+        static std::vector<ChatCommand> rbacAccountCommandTable =
+        {
+            { "list",   rbac::RBAC_PERM_COMMAND_RBAC_ACC_PERM_LIST,   true, &HandleRBACPermListCommand,    "" },
```

Comment:
For old chat commands use `SEC_` and other

Reply:
? Those are rbac commands.

Maybe I am missunderstanding you?

Reply:
```cpp
{ "list",   SEC_ADMINISTRATOR ,   true, &HandleRBACPermListCommand,    "" },
```

Reply:
No, you need to bind the rbac perm to the command, which is done like I did above. Otherwise rbac won't recognize it